### DISCUSSION TITLE
Welcome to Euler-Smea-Dyn-Sampler Discussions!

### DISCUSSION THREAD

**Discussion by Koishi-Star** [Koishi-Star/Euler-Smea-Dyn-Sampler#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vonwolfe** [comment]
Thanks for sharing this! I have git cloned it to my scripts folder, however euler smea isn't appearing in my samplers list when I run comfy UI. 
Do you have any advice about the steps I need to take to install it correctly? I can see on one of your updates that Capsadmin provided some help with this for installing on comfy UI?
Thank you!

**Reply by Koishi-Star** [reply]
Just like this:
![image](https://github.com/Koishi-Star/Euler-Smea-Dyn-Sampler/assets/66173435/b0ab0587-b720-4168-a954-65a13c1fb314)
![image](https://github.com/Koishi-Star/Euler-Smea-Dyn-Sampler/assets/66173435/749552e2-3b72-44d6-82d0-88c9a6be4ce3)

**Comment by fvwevewv** [comment]
该项目还在维护中吗，我发现这些采样器在使用同样的步数，种子，提示词，cfg强度，调度方式的情况下总是会生成不一样的图片，人物主体的动作是一致的但是衣服上的细节，背景，以及其他都是完全不一样的，这是为什么
